stdlib/public/RemoteInspection/TypeLowering.cpp

```diff
@@ -1602,6 +1602,12 @@ const TypeInfo *TypeConverter::getDefaultActorStorageTypeInfo() {
   return DefaultActorStorageTI;
 }
 
+const TypeInfo *TypeConverter::getRawUnsafeContinuationTypeInfo() {
+  // FIXME:  This is a bad approximation
+  return getReferenceTypeInfo(ReferenceKind::Strong,
+				 ReferenceCounting::Native);
```

Comment:
@mikeash Is this even close to right?  Is an UnsafeContinuation just a strong refcounted pointer?

Reply:
Answered below, but basically yes. Close enough for our purposes here.

Reply:
Whew!  I'll remove the FIXME then.